### DISCUSSION TITLE
Minimalist spectrograms

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#780] [Q&A] [answered]
Hello,

I just noticed in the settings "Minimalist spectrograms". What is it?

Thank you.

**Comment by ehpersonal38** [comment, marked as answer]
Hi @blogdemoi,

Minimalist spectrograms is an option within BirdNET-Pi that when enabled will make all spectrograms be generated without their usual axes and legends (such as the frequency axis on the left, and the dbFS legend on the right). I think it makes the UI look a little nicer which is why I added it. You can see an example below:

![xkeuSxr6av](https://user-images.githubusercontent.com/103586016/224197426-d5c0410f-c65b-4173-97e4-22b8c70c47af.png)

**Reply by blogdemoi** [reply]
Thank you.
